AzureMonitorAgent/agent.py

```diff
@@ -1718,6 +1718,13 @@ def parse_context(operation):
             logFileName = 'extension.log'
             hutil = HUtil.HandlerUtility(waagent.Log, waagent.Error, logFileName=logFileName)
             hutil.do_parse_context(operation)
+
+            # As per VM extension team, we have to manage rotation for our extension.log
+            # for now, this is our extension code, but to be moved to HUtil library.
+            if not os.path.exists('/etc/logrotate.d/azuremonitoragentextension'):      
```

Comment:
We need to remove this file on uninstall. Otherwise, an update running this path will not patch it.

Reply:
addressed it